frontend/src/pages/AccountSend.vue

```diff
@@ -72,7 +72,18 @@
       />
 
       <!-- Amount -->
-      <div>{{ $t('Send.amount') }}</div>
+      <div class="flex justify-between items-center">
```

Comment:
_(arbitrary comment location to start a thread)_

I haven't tested the actual sending yet, but one issue we'd run into is with the native token: Gas price might change from the time you fill in the max to the time you send your transaction, resulting in either (1) an insufficient balance error if gas price increases, or (2) dust leftover if gas price decreases. So for the native token we'll need a flag that indicates the user clicked the "Max" button and recompute the the actual max at the time of the send transaction. 

We also might want to throw an error if the amount changes by some threshold (+/- 1-2% maybe?) to prevent the user from sending a lot less than they expect if gas prices quickly change, but that might not be worth the added complexity for now.

I'm also ok with implementing this in a follow-up PR if we prefer, I'll defer to @garyghayrat or @apbendi on priority for that

Reply:
I'd say let's do it in a follow-up PR. For now @alexkeating feel free to open an issue for it.

Reply:
Yep agreed @garyghayrat.

Reply:
Great, tracking in https://github.com/ScopeLift/umbra-protocol/issues/378